### PR TITLE
Update to use Swift 3

### PR DESCRIPTION
**Pull request by kerrmarin**
Hi 👋 

This updates Alamofire-Decodable to Swift 3 🤖 

Things to watch out for:
- Alamofire has been updated to 4.0
- You have to use Cocoapods 1.1.0 (currently RC3 -`gem install cocoapods --pre`) because of Swift 3
- Decodable has been updated to 0.5.0

Breaking changes:
- Both public methods have been updated to use Alamofire 4.0 `DataResponse`

🚀 🎸

### CODE REVIEW

File: Alamofire-Decodable.podspec [hunk 1]
```diff
@@ -22,11 +22,11 @@ This pod adds the `responseDecodable` method to Alamofire's Request object to re
   s.source           = { :git => 'https://github.com/deanWombourne/Alamofire-Decodable.git', :tag => 'v' + s.version.to_s }
   s.social_media_url = 'https://twitter.com/deanWombourne'
 
-  s.ios.deployment_target = '8.0'
+  s.ios.deployment_target = '9.0'
```
Comment (deanWombourne) [inline Alamofire-Decodable.podspec:25]: Why no ios 8?

**Review by deanWombourne — COMMENTED** [review]
Looks good, though I don't know why you've dropped iOS 8?

File: Alamofire-Decodable.podspec (hunk 1 above)
Comment (kerrmarin) [reply Alamofire-Decodable.podspec:25]: Alamofire 4.0 doesn't support iOS 8 https://github.com/alamofire/alamofire#requirements 😢

Comment (deanWombourne) [reply Alamofire-Decodable.podspec:25]: Well, isn't that bloody annoying.

**Comment by deanWombourne** [comment]
Hey - I've pushed the latest master to trunk as version 2.0.0 - https://github.com/deanWombourne/Alamofire-Decodable/releases/tag/v2.0.0

**Comment by kerrmarin** [comment]
Awesome!

**Pull request merged**